src/wms/src/test/java/org/geoserver/wms/wms_1_1_1/GetLegendGraphicTest.java

```diff
@@ -108,13 +108,23 @@ public void testCustomLegend() throws Exception {
         BufferedImage expected = ImageIO.read( resource.file() );
         
         assertEquals( getPixelColor(expected,10,2).getRGB(), getPixelColor(image,10,2).getRGB() );
+
+        // test external image dimensions
```

Comment:
So the test checks the image size and not the value set in the setup... I suppose they are the same? Could be useful to have a test where they are not.